js/index.js

```diff
@@ -29,11 +21,69 @@ document.addEventListener("DOMContentLoaded", () => {
     "Project Management Systems (Jira, Asana, Wrike)",
     "Email Marketing (Klaviyo, Mailchimp)",
   ];
-  const skillsSection = document.getElementById('skills');
-  skillsList = skillsSection.querySelector('ul');
+  const skillsSection = document.getElementById("skills");
+  skillsList = skillsSection.querySelector("ul");
   for (let i = 0; i < skills.length; i++) {
     const skill = document.createElement("li");
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
   }
+
+
+
+  /////////////////////////////////////////
+  ///////////// MESSAGE FORM /////////////
+  ///////////////////////////////////////
+
+  // HIDE MESSAGES WHEN NONE//
+  document.getElementById("messages").style.display = "none";
+
+  // FORM //
+  const messageForm = document.getElementsByName("leave_message")[0];
+  messageForm.addEventListener("submit", (e) => {
+    e.preventDefault();
+
+    // USER INFO //
+    const usersName = e.target.usersName.value;
+    const usersEmail = e.target.usersEmail.value;
+    const usersMessage = e.target.usersMessage.value;
```

Comment:
When you have repetitive assignments, destructuring assignment is one option that can make it more concise.